lib/checkout.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/services.dart';
+import 'package:razorpay_flutter/upi_turbo.dart';
+
+class CheckOut{
```

Comment:
@shibu000 Why was this class created? We already have razorpay_flutter.dart. Can we not reuse the same class?

Reply:
@vivekshindhe The main reason is to follow the same function signature that we have approved in the API council review to expose it to the merchant.

Reply:
This is not it. The naming convention is wrong. And we will continue to use whatever class already exists. the name checkout is of the class in that SDK. It's not mandatory to use it. razorpay.upiTurbo.{funcName} is also reviewed approved. Please change this. We cannot ask the merchant to initialize another object. This beats the purpose of using it as a part of the base Checkout SDKs.